homeassistant/brands/sonos.json

```diff
@@ -0,0 +1,5 @@
+{
+  "domain": "sonos",
+  "name": "Sonos",
+  "integrations": ["sonos"]
```

Comment:
We currently do not support/accept single integration brands yet.

Reply:
I based this on the Mazda brand that currently only have a single integration: https://github.com/home-assistant/core/blob/5a51738b2f44b669f05cd366811caea00fe3978a/homeassistant/brands/mazda.json#L1-L5

So I'm not sure what is wrong here?

Reply:
The Mazda brand PR that was merged, has a comment open and is pending to be reverted. That one should not have been merged.

Reply:
Ah okay.

Do you guys prefer to leave PRs open or should I close this for now?

Reply:
I'm not sure yet. I'm currently working on a proposal/figuring out what to do with these cases. Especially the "supported_brands" cases (and other open ends). Until a fitting proposal has been made and approved, I suggest not to currently pursue this PR.

Reply:
Sounds like a plan then. I'll leave it open from my side but feel free to close it at any time if need be. 👍🏻